recipes/r-animation/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = '2.5' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-animation
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: animation_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/animation_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/animation/animation_{{ version }}.tar.gz
+  sha256: b232fef1b318c79710e5e1923d87baba4c85ffe2c77ddb188130e0911d8cb55f
+
+build:
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
```

Comment:
This package has many system requirements. From the [CRAN entry](https://cran.r-project.org/package=animation):

> SystemRequirements: ImageMagick (http://imagemagick.org) or GraphicsMagick (http://www.graphicsmagick.org) or LyX (http://www.lyx.org) for saveGIF(); (PDF)LaTeX for saveLatex(); SWF Tools (http://swftools.org) for saveSWF(); FFmpeg (http://ffmpeg.org) or avconv (https://libav.org/avconv.html) for saveVideo()

Reply:
conda-forge has ImageMagick and FFmpeg available, so those are easy to add as dependencies.

https://github.com/conda-forge/imagemagick-feedstock
https://github.com/conda-forge/ffmpeg-feedstock

ImageMagick is not available for Windows, so you'd need to add `  skip: true  # [win]`.

Not sure about LaTeX or SWF tools.

Reply:
Oh yes sorry about that ! Don't you think that adding texlive-core would work out ? I guess it's going to be a bit heavy though...

Reply:
Yes! Good call. Adding `texlive-core` as a dependency will provide `pdflatex`.

Reply:
It looks like in principle that SWF Tools could be packaged with conda, but I don't know how much work that would take.

http://swftools.org/download.html

Do you use the function `saveSWF()` at all? If not, then I think it'd be fine if you skip that dependency for now. If someone else really needs it, they can create a conda package and then update the recipe for r-animation.